### ISSUE TITLE
download failed: the underlying connection was closed: could not establish trust relationship for the ssl/tls secure channel

### ISSUE THREAD

**Issue by vanydakid** [n00mkrad/flowframes#300]
i can't install flowframes? i don't know how to fix this error plz help

**Comment by Goodboi32** [comment]
I have the same issue, it was working like 4 days ago but now i tried to reinstall and it wont install for me no more @n00mkrad

**Comment by TrilentGH** [comment]
I have same problem..

**Comment by kestyfresh** [comment]
I wanted to try it today for the first time and have this problem as well...

**Comment by n00mkrad** [comment]
Works again? Seems like the SSL cert expired

**Comment by TrilentGH** [comment]
> Works again? Seems like the SSL cert expired

yea it works again ;)

**Comment by kestyfresh** [comment]
works on my end too, thx!

**Comment by imDevinder** [comment]
It stopped working again. Please, fix this issue!😢

**Comment by OneAnimeGuy** [comment]
It stopped wokring. Today is April 21 at the time of writing. I was installing flowframe but this error popped out.
Can someone please tell me how to fix it?
![Screenshot 2024-04-21 161338](https://github.com/n00mkrad/flowframes/assets/167680966/888330a2-490b-4463-a0ea-370626dfc6b3)
![Screenshot 2024-04-21 161413](https://github.com/n00mkrad/flowframes/assets/167680966/70923873-44fd-469b-9a58-92cd439246bb)

**Comment by bl0oby** [comment]
yeah, i was trying to download it just now and i've gotten the same error, did the SSL cert expire again? @n00mkrad

**Comment by MarrowGG** [comment]
@n00mkrad I think the SSL cert expired because I'm getting this issue, can you please check

**Comment by imDevinder** [comment]
Bro use VPN (Proton Free Vrs.), I used it and it solved the problem.. I am using it !!

**Comment by bl0oby** [comment]
> Bro use VPN (Proton Free Vrs.), I used it and it solved the problem.. I am using it !!

yo!!! thanks so much brother you're a legend!

**Comment by Valera005** [comment]
> Bro use VPN (Proton Free Vrs.), I used it and it solved the problem.. I am using it !!

Doesn't work for me for some reason, still get the problem

**Comment by bl0oby** [comment]
> Doesn't work for me for some reason, still get the problem

look, i use this vpn called windshield (search it up on google and download the desktop version), and create an account (you don't need an email, username and password only, however, im not so sure but i think you get more data if you put your email in the sign up page) once everything was set up, i set my vpn location to dallas, closed everything down apart from flowframes (so it doesn't consume too much out of the vpn free data), and that'll allow you do download from the St. Louis server, once the download starts to work you can either turn off the vpn or just leave it on until it's fully downloaded. Hope this helps!

**Comment by n00mkrad** [comment]
Should be fixed, need into looking automation of the cert renewal

**Comment by dobkun** [comment]
it expired again

**Comment by Niko7723** [comment]
I'm having the same problem. I've tried everything, but it didn't work. I used several different VPNs. I changed the DNS server. I changed my network connection. I tried everything, but it still won't start downloading.

**Comment by n00mkrad** [comment]
So sorry.

Fixed it, and automatic renewal should finally be fixed as well now.
